### DISCUSSION TITLE
NetCDF4 to GeoTIFF

### DISCUSSION THREAD

**Discussion by aswinkokkat** [opengeos/leafmap#313] [Q&A] [answered]
Using the leafmap tutorial 52 [https://leafmap.org/notebooks/52_netcdf/](url) i was able to generate one tiff. In my NetCDF4 file i have monthly data (Variable: pgwuse) from 1901 to 2016 (116 * 12 = 1392). Is it possible to export monthly data to geotiff ?

import leafmap
filename ='NetCDF.nc4'
data = leafmap.read_netcdf(filename, decode_times=False)
data

xarray.Dataset
Dimensions:
lat: 360lon: 720time: 1392
Coordinates:
lat
(lat)
float32
89.75 89.25 88.75 ... -89.25 -89.75
lon
(lon)
float32
-179.8 -179.2 ... 179.2 179.8
time
(time)
float32
0.0 1.0 2.0 ... 1.39e+03 1.391e+03
Data variables:
pgwuse
(time, lat, lon)
float32
...
Indexes:
lat
PandasIndex
lon
PandasIndex
time
PandasIndex
Attributes:
title :
Potential total consumptive water use from groundwater - WaterGAP

tif = 'pgwuse.tif'
leafmap.netcdf_to_tif(filename, tif, variables=['pgwuse'], shift_lon=True, decode_times=False)
'C:\\Users\\kokka\\Desktop\\NewFolder\\pgwuse.tif'

**Comment by giswqs** [comment, marked as answer]
You can use rioxarray to convert netCDF to GeoTIFF
https://gis.stackexchange.com/a/329141

**Reply by aswinkokkat** [reply]
Thank you very much for the reply, I am getting an error while giving output path 
"xds["anag"].rio.to_raster('C:\\Users\\NGRI\\Desktop\\GWL')"
---> 75     raise ValueError("Unable to detect driver. Please specify driver.")

ValueError: Unable to detect driver. Please specify driver.

**Reply by giswqs** [reply]
Add output file extension.tif
